resources/views/maps/groups/_form.blade.php

```diff
@@ -16,10 +16,13 @@
 
 @php 
     $options = $map->groupPositionOptions();
+    if (!empty($model->position) && $model->position == array_key_last($options) - 1) {
```

Comment:
Would be cleaner to have this in the `groupPositionOption()` function with an optional param for the current position, avoiding logic in the views as much as possible. Also a comment describing what this does would be nice because I had no idea by just reading the code :)